doc/src/guides/solving.md

```diff
@@ -0,0 +1,25 @@
+# Solve Using Python SymPy
+
+Python package SymPy can symbolically solve equations, differential equations, 
+linear equations, nonlinear equations, matrix problems, inequalities, 
+diophantine equations, and integrals. SymPy can also solve numerically.
+
+Learn how to use SymPy computer algebra system to:
+
+- [Solve an equation]() or [solve a system of equations algebraically]()
+- [Solve an equation numerically]()
+- [Solve a partial differential equation algebraically]()
+- [Solve ordinary differential equations algebraically]()
+- [Solve a system of linear equations algebraically]()
+- [Solve a system of nonlinear equation algebraically]()
+- [Solve a matrix problem algebraically]()
+- [Solve an inequality]() or [solve a system of inequalities]() algebraically
+- [Solve a polynomial algebraically]()
+- [Solve a diophantine equation algebraically]()
+- [Solve an integral symbolically]()
```

Comment:
In the glossary I wrote

>   SymPy generally uses the word "solve" to mean equation solving in this
    sense. It is not used in the sense of "solving a problem". For instance,
    one would generally prefer to use a word like "compute" or "evaluate" to
    refer to the computation of an integral using a function like
    integrate.

https://github.com/sympy/sympy/pull/23293/files#diff-f3a7e8594f4f56ce385b5d650df921ee5cb516cc7b94db0b0c42679a70bdd6a8R558

So I think we should stick with that and avoid using "solve" to mean integration here. We can also add some similar text to this page to help people understand this distinction.

Reply:
That makes sense from a SymPy nomenclature standpoint.

To check how that might affect search engine optimization (SEO) and presumably visitor [information architecture](https://www.usability.gov/what-and-why/information-architecture.html) (how content is organized, structured, and labeled so visitors can find what they need), I investigated further the [year's worth of Google Search Console data](https://docs.google.com/spreadsheets/d/1zuQDU1Mj8Ua8zAAxRu5hr7b23wCRlIJ2cOK6FPMnJq0/edit?usp=sharing) and found that "sympy evaluate integral" has somewhat more impressions than "sympy solve integral", 146 vs. 103, so "evaluate" is also preferable from an SEO standpoint. (There are no search terms in the top 1000 including "compute" and "integral".)

So I'll change "Solve an integral" to "Compute an integral".

I reviewed the list of solving types and didn't notice any others where "solve" conflicts with the definition in the glossary. If anyone believes there are others, please let me know.

Reply:
I am confused. Wouldn't it help with searching and be more consistent with common usage, while also being consistent with Sympy vocabulary to do something like this:

"Evaluate (solve) an integral". 

Why use a term (compute) that people use rarely?

Reply:
Oops! Thanks. I had "compute" on the brain. I changed it to "evaluate" in the pull request, and edited my comment above so someone reviewing my comment later won't notice the same mistake.